Update protect-your-global-administrator-accounts.md (#751)
MicrosoftDocs/OfficeDocs-Enterprise: Enterprise/protect-your-global-administrator-accounts.md

# Protect your Office 365 global administrator accounts

> [!NOTE]

> > The Global admin account can be created without any adding any license.

Reviewer: ```suggestion
> A global administrator account can be created without adding any licenses.
```
